scripts/install

```diff
@@ -93,29 +101,28 @@ ynh_script_progression --message="Configuring NGINX web server..." --weight=1
 ynh_add_nginx_config
 
 #=================================================
-# CREATE DEDICATED USER
+# SPECIFIC SETUP
 #=================================================
-ynh_script_progression --message="Configuring system user..." --weight=1
+# ADD A CONFIGURATION
+#=================================================
+ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-# Create a system user
-ynh_system_user_create --username="$app" --home_dir="$final_path"
+ynh_add_config --template="config.json5" --destination="$final_path/config.json5"
+
+chmod 400 "$final_path/config.json5"
+chown $app:$app "$final_path/config.json5"
 
-#=================================================
-# SPECIFIC SETUP
 #=================================================
 # BUILD YARN DEPENDENCIES
 #=================================================
 ynh_script_progression --message="Building Yarn dependencies... This may take several minutes for a first install." --weight=6
 
-chown -R "$app":"$app" "$final_path"
-
 pushd "$final_path" || return 1
     ynh_use_nodejs ; set "${ynh_node_load_PATH:?}"
-    sudo -u "$app" env "$ynh_node_load_PATH" yarn
-    sudo -u "$app" env "$ynh_node_load_PATH" yarn build
-    sudo -u "$app" env "$ynh_node_load_PATH" yarn install --production --ignore-scripts --prefer-offline
-
-    yarn_bin_dir="$(sudo -u "$app" env "$ynh_node_load_PATH" yarn global bin)"
+    sudo -u $app env "$ynh_node_load_PATH" yarn
+    sudo -u $app env "$ynh_node_load_PATH" yarn build
+    sudo -u $app env "$ynh_node_load_PATH" yarn install --production --ignore-scripts --prefer-offline
+    ynh_secure_remove --file="$final_path/.cache"
```

Comment:
.cache will contain build files that will accelerate upgrades.

Reply:
I know but it's renewed during next yarn install ... and when I tested it, it was like 350Mo....
Is it better to have smaller backups or quicker upgrades ?

Reply:
Ah I didn't realize this was backed up. Isn't there any way to prevent this .cache dir to be backed up ?